mffpy/bin_writer.py

```diff
@@ -157,15 +156,19 @@ def __init__(self, sampling_rate: int, mffdir: str, data_type: str = 'EEG'):
         * **`data_type`**: name of the type of signal.
 
         * **`mffdir`**: directory of the mff recording to stream data to.
-        
-        Note: Because we are streaming the recording to disk, the folder into which it
-        is to be saved must have been created prior to the initialization of this class.
+
+        **Notes**
+
+        Because we are streaming the recording to disk, the folder into which
+        it is to be saved must have been created prior to the initialization of
+        this class.
         """
-        
+
         super().__init__(sampling_rate, data_type)
-        self.stream = FileIO(join(mffdir, self.default_filename), mode='w')
+        filename = self.default_filename_fmt % 1
```

Comment:
Again, here I would determine `filename` from `self.data_type`.

Reply:
What if someone only writes out `PNSData`?  Why do you suspect that the ordering of EEG in signal1.bin and PNSData in signal2.bin matters?  Is there a place in the file description wherein the information is implicitely accessed?

I'll add a warning however.

Reply:
Added a warning.

Reply:
So I did some testing and my suspicion was correct that if signal1.bin/info1.bin is PNSData and signal2.bin/info2.bin is EEG, then the MFF will crash Net Station. True, the datatype is contained in the info file, but Net Station seems to require the files to be named according to the convention. This was my main concern.

I think forcing this file naming convention would be easy enough to implement and would prevent users from writing "broken" MFFs in this regard.

Reply:
Would we ever want to write PNSData with this `StreamingBinWriter`? If so, we probably want the `filename` to be allowed to be set to signal2.bin.

Reply:
Would we ever want to write PNSData with this `StreamingBinWriter`? If so, we probably want the `filename` to be allowed to be set to signal2.bin.

Reply:
`StreamingBinWriter` is not of concern in this PR.